app/controllers/listings_controller.rb

```diff
@@ -132,27 +132,24 @@ def create_google_place_variables(google_place_id)
     return if Rails.env.test?
     @google_places_client = GooglePlaces::Client.new(ENV["GOOGLE_MAPS_API_KEY"])
     @google_place = @google_places_client.spot(google_place_id)
-    @reviews = @google_place.reviews[0..4]
+    @reviews = @google_place.reviews.first(5)
   end
 
   def set_listings_and_coordinates
     if params[:location].present?
       @listings = Listing.near(params[:location], 5)
 
-      lat_long = Geocoder.coordinates params[:location]
-      gon.centerPoint = { latitude: lat_long[0], longitude: lat_long[1], zoom: 13 }
+      latitude, longitude = Geocoder.coordinates params[:location]
+      gon.centerPoint = { latitude: latitude, longitude: longitude, zoom: 13 }
     else
       gon.centerPoint = center_point_from_ip_address
       @listings = Listing.near([gon.centerPoint[:latitude], gon.centerPoint[:longitude]], 5)
     end
 
-    gon.coordinates = @listings.map { |listing| [listing.lat, listing.long] }
+    gon.coordinates = @listings.map(&:coordinates)
   end
```

Comment:
This syntax always hurts my head to remember what's happening. But I see it enough that I probably just need to remember it.

Reply:
Ha, yeah, it's pretty common.